Modules/Loadable/Markups/MRML/vtkMRMLMeasurementArea.cxx

```diff
@@ -79,9 +79,9 @@ void vtkMRMLMeasurementArea::Compute()
     }
   else if (planeNode)
     {
-    double size[2] = { 0.0 };
-    planeNode->GetSize(size);
-    area = size[0] * size[1];
+    double size_world[2] = { 0.0 };
```

Comment:
This kind of initialization is somewhat misleading, as it may suggest that it affects all values in the vector, while it only changes the first value (the others are set to 0.0).

Examples:

```
x[3] => 6.95324e-310 4.94066e-324 6.95324e-310
x[3] = {} => 0 0 0
x[3] = {1.0}; => 1 0 0
x[3] = {1.0, 1.0, 1.0}; => 1 1 1
```

https://onlinegdb.com/0w5Ry2JbL

So, if we initialize to 0.0 then we should use `{}` or if we want to be more explicit then to `{0.0, 0.0}`. Since explicitly writing down all the values is harder to misinterpret, it works for all values (not just 0.0), and we do this at many places already, it is probably better to use `{0.0, 0.0}` here, too.